Source/ACE.Entity/Position.cs

```diff
@@ -172,13 +172,13 @@ public Position(float northSouth, float eastWest)
             uint baseX = (uint)(eastWest + 0x400);
             uint baseY = (uint)(northSouth + 0x400);
 
-            if (baseX < 0 || baseX >= 0x7F8 || baseY < 0 || baseY >= 0x7F8)
+            if (baseX >= 0x7F8 || baseY >= 0x7F8)
                 throw new Exception("Bad coordinates");  // TODO: Instead of throwing exception should we set to a default location?
```

Comment:
I'm ok with the throw.  It's a legit bug.  Throw early and ugly.  Quietly swallowing errors allows bugs to become sneaky.